pkg/runtime/grpc/grpc-runtime.go

```diff
@@ -141,7 +158,14 @@ func (r *Runtime) ParamDescs() params.ParamDescs {
 }
 
 func (r *Runtime) GlobalParamDescs() params.ParamDescs {
-	p := params.ParamDescs{}
+	p := params.ParamDescs{
+		{
+			Key:          ParamConnectionTimeout,
+			Description:  "Maximum time to establish a connection to remote target in seconds",
+			DefaultValue: fmt.Sprintf("%d", ConnectTimeout),
+			TypeHint:     params.TypeInt,
```

Comment:
nit: TypeUint?

Reply:
`TypeDuration` parses a different format, it expects something like `3s` for 3 seconds (instead of just a `3` for an int). I think for such small durations having to use the `3s` confuses more than it helps, that's why I chose to stay with ints.

Reply:
btw, I just noticed that we have a `--request-timeout string         The length of time to wait before giving up on a single server request. Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h). A value of zero means don't timeout requests. (default "0"` parameter on kubectl-gadget (probably coming from the common kubernetes flags). Do you think these two parameters could be combined?

Reply:
Hmm, I don't know. It would probably be okay, but I'd not be fine with the default value of `0`. But then again, I like to have it separated because both things are really only useful in edge cases where it might be actually important to control the values independently.